### DISCUSSION TITLE
CTkTabview

### DISCUSSION THREAD

**Discussion by VovkVital** [TomSchimansky/CustomTkinter#958] [Q&A] [answered]
I can't figure it out how to use "command argument" for a tabveiw on the added tabs; it gives me an error after  I use it in.
 Im adding "command" to  `"********".tab("My tab").configure(command = "Function")` , an error message states `__**"ValueError: ['command'] are not supported arguments. Look at the documentation for supported arguments._`"**_.

Is that possible to have a function called when you switch between added tabs in CTkTabveiw?

**Comment by VLTNOgithub** [comment, marked as answer]
Add the argument in the statement where you make the tabview. Ex: `tabview = customtkinter.CTkTabview(self, width=300, height=250, command=onTabChanged)`
And have a function before it like this: `def onTabChanged():
            if tabview.tab("My Tab"):
                #Do something if the active tab is "MyTab"`
